packages/react-components/src/components/UnsupportedBrowserVersion.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(unsupported-browser) */
+import { _pxToRem } from '@internal/acs-ui-common';
+import React from 'react';
+/* @conditional-compile-remove(unsupported-browser) */
+import { UnsupportedEnvironment } from './UnsupportedEnvironment';
+
+/**
+ * Strings for UnsupportedBrowser component
+ *
+ * @beta
+ */
+export interface UnsupportedBrowserVersionStrings {
+  /** String for the primary text */
+  primaryText: string;
+  /** String for the secondary text */
+  secondaryText: string;
+  /** String for the help link */
```

Comment:
Is this the URL that the link points to, or the text that is shown for the link?

If it is the URL, perhaps this should be call `moreHelpUrl`.

And in either case, the docstring could be more helpful.